app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/LayoutCalculator.kt

```diff
@@ -0,0 +1,110 @@
+package nerd.tuxmobil.fahrplan.congress.schedule
+
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import info.metadude.android.eventfahrplan.commons.logging.Logging
+import info.metadude.android.eventfahrplan.commons.temporal.Moment
+import nerd.tuxmobil.fahrplan.congress.models.Lecture
+import org.threeten.bp.Duration
+import kotlin.collections.List
+import kotlin.collections.Map
+import kotlin.collections.indices
+import kotlin.collections.mutableMapOf
+import kotlin.collections.set
+
+data class LayoutCalculator(val logging: Logging = Logging.get(),
+                            val standardHeight: Int) {
+    private val DIVISOR = 5
+    private val MILLIS_PER_MINUTE = 60000
+
+    companion object {
+        val LOG_TAG = "LayoutCalculator"
+    }
```

Comment:
:mag: Please move the constants into the companion object and mark them as `const` and let the object be  `private`:

``` kotlin
private companion object {
    const val LOG_TAG = "LayoutCalculator"
    const val DIVISOR = 5
    const val MILLIS_PER_MINUTE = 60000
}
```